akka-stream/src/main/scala/akka/stream/SslTlsOptions.scala

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright (C) 2015-2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream
```

Comment:
Which inconsistencies do you mean? Akka Streams is completely built like this. The Akka HTTP model is different, granted, but we cannot retroactively apply that principle to Streams now.